packages/hoppscotch-backend/src/admin/admin.resolver.ts

```diff
@@ -269,6 +70,27 @@ export class AdminResolver {
     return invitedUser.right;
   }
 
+  @Mutation(() => Boolean, {
+    description: 'Revoke a user invite by invitee email',
+  })
+  @UseGuards(GqlAuthGuard, GqlAdminGuard)
+  async revokeUserInvitationByAdmin(
+    @GqlAdmin() adminUser: Admin,
+    @Args({
+      name: 'inviteeEmail',
+      description: 'Invite Email',
+      type: () => ID,
+    })
+    inviteeEmail: string,
+  ): Promise<boolean> {
+    const invite = await this.adminService.revokeUserInvite(
```

Comment:
I can see this also becoming a bulk actionable feature in the future, shall we make it accept an array or invitee emails. I think we should do it this way since its a very small change.

Reply:
Done